File: api/src/main/java/com/ruby/rubia_server/core/enums/MessagingProvider.java
package com.ruby.rubia_server.core.enums;

public enum MessagingProvider {
    Z_API("Z-API"),
    TWILIO("Twilio"),
    WHATSAPP_BUSINESS_API("WhatsApp Business API"),
    MOCK("Mock Provider");

    private final String displayName;

    MessagingProvider(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: api-dotnet/Rubia.Server/Enums/MessagingProvider.cs
using System.ComponentModel;

namespace Rubia.Server.Enums;

public enum MessagingProvider
{
    [Description("Z-API")]
    ZApi,
    
    [Description("Twilio")]
    Twilio,
    
    [Description("WhatsApp Business API")]
    WhatsAppBusinessApi,
    
    [Description("Mock Provider")]
    Mock
}